CONTRIBUTING.md

```diff
@@ -24,7 +24,6 @@ You can also join our [Community Slack channel](https://community.nginx.org/join
 
 Please reserve Github issues for feature requests and bugs rather than general questions.
```

Comment:
```suggestion
Please reserve GitHub issues for feature requests and bugs rather than general questions.
```

Noticed this while going through the PR.  Out of curiosity, is there an expected channel/guidance on where someone should go for general questions? Right now people are being told "Not here", but not told where they should be, instead.

Reply:
Decided to review the whole document to see if it was covered elsewhere - TIL Discussions exist.